spec/facility_spec.rb

```diff
@@ -1,26 +1,142 @@
-      expect(@facility.phone).to eq('(720) 865-4600')
-      expect(@facility.services).to eq([])
+
+  describe "#initialize" do
+    it "can initialize" do
+      expect(@facility_1).to be_an_instance_of(Facility)
+      expect(@facility_1.name).to eq('DMV Tremont Branch')
+      expect(@facility_1.address).to eq('2855 Tremont Place Suite 118 Denver CO 80205')
+      expect(@facility_1.phone).to eq('(720) 865-4600')
+      expect(@facility_1.daily_hours).to eq('Mon, Tue, Thur, Fri  8:00 a.m.- 4:30 p.m. / Wed 8:30 a.m.-4:30 p.m.')
+      expect(@facility_1.observed_holidays).to eq("Independence Day (07/04/22), Labor Day (09/05/22), Columbus Day (10/10/22), Veterans Day (11/11/22), Thanksgiving (11/24/22), Christmas Day Observed (12/26/22), New Year's Day Observed (01/02/23), Martin Luther King Day (01/16/23), Lincoln’s Birthday (02/13/23), President’s Day (02/20/23), Truman’s Birthday (05/08/23), Memorial Day (05/29/23), Juneteenth (06/19/2023), Independence Day (07/04/23), Labor Day (09/04/23), Columbus Day (10/09/23), Veterans Day (11/10/23), Thanksgiving (11/23/23), Christmas Day Observed (12/25/23)")
+      expect(@facility_1.services).to eq([])
+    end
+
+    it "starts with no registered vehicles and no collected fees" do
+      expect(@facility_1.registered_vehicles).to eq([])
+      expect(@facility_1.collected_fees).to eq(0)
     end
   end
 
   describe '#add service' do
-    it 'can add available services' do
-      expect(@facility.services).to eq([])
-      @facility.add_service('New Drivers License')
-      @facility.add_service('Renew Drivers License')
-      @facility.add_service('Vehicle Registration')
-      expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+    it "can add available services" do
+      expect(@facility_1.services).to eq([])
+      @facility_1.add_service('New Drivers License')
+      @facility_1.add_service('Renew Drivers License')
+      @facility_1.add_service('Vehicle Registration')
+      expected = ['New Drivers License', 'Renew Drivers License', 'Vehicle Registration']
+      expect(@facility_1.services).to eq(expected)
+    end
+    it "can't execute services not available" do
+    expect(@facility_2.services).to eq([])
+    expect(@facility_2.register_vehicle(@bolt)).to eq (nil)
+    expect(@facility_2.registered_vehicles).to eq([])
+    expect(@facility_2.collected_fees).to eq(0)
+    expect(@facility_1.administer_written_test(@registrant_1)).to eq(false)
+    expect(@facility_1.administer_road_test(@registrant_2)).to eq(false)
+    expect(@facility_1.renew_drivers_license(@registrant_3)).to eq(false)
+    end
+  end
+
+  describe "#register a vehicle" do
+    it "can register a vehicle" do
+      @facility_1.add_service('Vehicle Registration')
+      expect(@cruz.registration_date).to eq (nil)
+      expect(@facility_1.registered_vehicles).to eq []
+      expect(@facility_1.collected_fees).to eq(0)
+
+      @facility_1.register_vehicle(@cruz)
```

Comment:
Yes for the white spaces. Feel free to use more white spaces in your code to organize chunks of a test, helps the eyes.